template.distillery/sass!os.scss

```diff
@@ -276,4 +276,13 @@ a.os-page-header-app-name {
             }
         }
     }
+}
+
+.os-avatar {
+    margin-right: 15px;
+    width: 35px;
+}
+
+.ot-dr-left .os-avatar {
+    margin-left: 16px;
```

Comment:
This line is here to have the same space on the left than the elements in the drawer.